src/Employee/Employee.java

```diff
@@ -0,0 +1,36 @@
+package Employee;
+
+public class Employee {
+
+    private int id;
+    private String surname;
+    private double salary;
+
+    public Employee() {
```

Comment:
It should be removed for optimization because it's constuctor without parameters and it is the same as a default constructor

Reply:
Ok, it's just lesson example